src/chainparams.cpp

```diff
@@ -451,9 +443,12 @@ class CRegTestParams : public CChainParams {
         nModulusV2StartBlock = 130;
         nModulusV1MempoolStopBlock = 135;
         nModulusV1StopBlock = 140;
-        nMTPSwitchTime = INT_MAX;
+        nMTPSwitchTime = 1;
         nDifficultyAdjustStartBlock = 0;
         nFixedDifficulty = 0;
+
+        if(genesis.IsMTP())
```

Comment:
genesis block is never MTP

Reply:
Yes, realized that but left the code here by mistake. Removing.